src/ztrsm_LLN.jdf

```diff
@@ -26,6 +26,8 @@ alpha     [type = "dplasma_complex64_t"]
 descA     [type = "const parsec_tiled_matrix_t*"]
 descB     [type = "parsec_tiled_matrix_t*"]
 
+cuda_handles_infokey    [type = "int" hidden = on default = PARSEC_INFO_ID_UNDEFINED ]
```

Comment:
Why do we need both ? Can't we just use `handles_infokeys` and then populate it with the correct values in the wrapper ? Or do you expect the CUDA and HIP versions to be used in the context of the same application ?

Btw, where is the `cuda_handles_infokey` used ?

Also, why don't you just set the default to `parsec_info_lookup(&parsec_per_stream_infos, "DPLASMA::HIP::HANDLES", NULL)` ?

Reply:
Jumping the gun, I wanted to change everything to use cublass_v2 so that would entail that we have a stream info for cuda as well. I'll move this to a later PR.